### PR TITLE
feat(swap): add swap page with tabs

### PR DESCRIPTION
**Pull request by nistadev**
**Screenshots**
image

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscirpt] Jira Ticket: **[Create the SWAP page in revamp with two empty tabs and it should be available only in dev mode for now](https://emurgo.atlassian.net/browse/YOEXT-647)**

[](__JIRA_COMMENT_FLAG)

File: packages/yoroi-extension/app/containers/swap/SwapPageContainer.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+import SidebarContainer from '../SidebarContainer';
+import NavBarTitle from '../../components/topbar/NavBarTitle';
+import NavBarContainerRevamp from '../NavBarContainerRevamp';
+
+export type GeneratedData = typeof SwapPageContainer.prototype.generated;
+
+type Props = {|
+  ...InjectedOrGenerated<GeneratedData>,
+  +children?: Node,
+|};
```
Comment (neuodev) [inline packages/yoroi-extension/app/containers/swap/SwapPageContainer.js:29]: Empty line please 🙂 
```suggestion
|};

```

File: packages/yoroi-extension/app/containers/swap/categories/OrdersPage.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        },
+      },
+      actions: {
+        explorers: {
+          updateSelectedExplorer: { trigger: actions.explorers.updateSelectedExplorer.trigger },
+        },
+      },
+      canRegisterProtocol: environment.userAgentInfo.canRegisterProtocol,
+    });
+  }
```
Comment (neuodev) [inline packages/yoroi-extension/app/containers/swap/categories/OrdersPage.js]: All these values are not needed. I think you didn't do it on purpose. You forgot to delete it. 

```suggestion
  @computed get generated(): {|
    actions: {||},
    stores: {||},
  |} {
    if (this.props.generated !== undefined) {
      return this.props.generated;
    }
    if (this.props.stores == null || this.props.actions == null) {
      throw new Error(`${nameof(SwapOrdersPage)} no way to generated props`);
    }    
    return Object.freeze({
      stores: {},
      actions: {},
    });
  }
```

File: packages/yoroi-extension/app/containers/swap/categories/OrdersPage.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import { isCardanoHaskell } from '../../../api/ada/lib/storage/database/prepackaged/networks';
+import { SelectedExplorer } from '../../../domain/SelectedExplorer';
+import { trackUriPrompt } from '../../../api/analytics';
+import ExplorerSettings from '../../../components/settings/categories/general-setting/ExplorerSettings';
+import UriSettingsBlock from '../../../components/settings/categories/general-setting/UriSettingsBlock';
+import registerProtocols from '../../../uri-protocols';
+import environment from '../../../environment';
+import LocalizableError from '../../../i18n/LocalizableError';
+import NoWalletMessage from '../../wallet/NoWalletMessage';
+
```
Comment (neuodev) [inline packages/yoroi-extension/app/containers/swap/categories/OrdersPage.js]: Not all these imports are needed. Please include only needed imports.

File: packages/yoroi-extension/app/containers/swap/categories/OrdersPage.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        wallets: {
+          selected: stores.wallets.selected,
+        },
+      },
+      actions: {
+        explorers: {
+          updateSelectedExplorer: { trigger: actions.explorers.updateSelectedExplorer.trigger },
+        },
+      },
+      canRegisterProtocol: environment.userAgentInfo.canRegisterProtocol,
```
Comment (neuodev) [inline packages/yoroi-extension/app/containers/swap/categories/OrdersPage.js]: Please remove these stores and actions as you don't need them.

File: packages/yoroi-extension/app/containers/swap/categories/SwapPage.js [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+export default class SwapPage extends Component<InjectedOrGenerated<GeneratedData>> {
+  static contextTypes: {| intl: $npm$ReactIntl$IntlFormat |} = {
+    intl: intlShape.isRequired,
+  };
+
+  render(): Node {
+    return <>Swap form here</>;
+  }
+
+  @computed get generated(): {|
```
Comment (neuodev) [inline packages/yoroi-extension/app/containers/swap/categories/SwapPage.js]: Should be deleted as well.

**Review by neuodev — CHANGES_REQUESTED** [review]

File: packages/yoroi-extension/.flowconfig [hunk 6]
```diff
@@ -31,7 +31,7 @@ deprecated-utility=error
 
 [options]
 types_first=true
-include_warnings=false
+include_warnings=true
```
Comment (zuzunker) [inline packages/yoroi-extension/.flowconfig]:
```suggestion
include_warnings=false
```

**Review by zuzunker — DISMISSED** [review]

**Pull request merged**